pybikes/data/metro_bike_share.json

```diff
@@ -0,0 +1,17 @@
+{
+    "instances": [
+        {
+            "tag": "metro_bike_share",
```

Comment:
Why do we use a tag that breaks with how other tags are named?

`metro-bike-share`